app/api/user.py

```diff
@@ -0,0 +1,96 @@
+from app import db
+from app.models import Users, Account
+from flask import Blueprint, request, jsonify
+from flask_jwt_extended import jwt_required, current_user
+
+bp = Blueprint('user', __name__)
+
+
+@bp.route('/invite', methods=['POST'])
+@jwt_required()
+def invite():
+    data = request.get_json()
+
+    attributes = data.get('data', {}).get('attributes', {})
+    new_username = attributes.get('new_username')
+
+    if not attributes or not new_username:
+        return jsonify({"errors": [{
+            "status": "400",
+            "detail": "Invalid syntax"
+        }]}), 400
```

Comment:
All endpoints contain a lot of code that do payload validation. Find a way to make it easier, e.g. use pydandic for api payload validation.